persistence/commons/src/main/java/org/apache/causeway/persistence/commons/integration/repository/RepositoryServiceDefault.java

```diff
@@ -99,6 +105,27 @@ public <T> T detachedEntity(final @NonNull T entity) {
         return factoryService.detachedEntity(entity);
     }
 
+    @Override
+    public <T> T execInBulk(Callable<T> callable, Class<?>... classes) {
```

Comment:
A unit test for this method would be worthwhile, I think.

Reply:
Changed an exisiting test to use bulk version.